Improve instructions for self-hosted Postgres live-sync (#4235)
timescale/docs: _partials/_livesync-console.md

## Limitations

* The source $PG instance must be accessible from the Internet.

>   Services hosted behind a firewall or VPC is not supported. It is in the

Reviewer: ```suggestion
  Services hosted behind a firewall or VPC are not supported. This functionality is on the roadmap. 
```

>   roadmap to support this in the future.

Reviewer: ```suggestion
```

> * Indexes (including Primary Key and Unique constraints) are not migrated by $SERVICE_LONG.

Reviewer: ```suggestion
* Indexes, including the primary key and unique constraints, are not migrated to the target $SERVICE_LONG.
```

>   We recommend that you create only the necessary indexes on the target $SERVICE_LONG depending on your query patterns.

Reviewer: ```suggestion
  We recommend that, depending on your query patterns, you create only the necessary indexes on the target $SERVICE_LONG.
```
